modules/sso/bz-1769831/install

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+mkdir -p /run/user/$UID
```

Comment:
Looks OK per the [corresponding `microdnf/librepo`](https://github.com/rpm-software-management/librepo/pull/178/files) change.